package.json

```diff
@@ -62,45 +62,44 @@
     "passport": "^0.4.0",
     "passport-custom": "^1.1.0",
     "passport-google-oauth20": "^2.0.0",
-    "query-string": "^6.8.2",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "timeago.js": "^4.0.0-beta.2"
+    "query-string": "^6.8.3",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "timeago.js": "^4.0.1"
```

Comment:
https://github.com/hustcc/timeago.js/releases/tag/v4.0.0 and https://github.com/hustcc/timeago.js/issues/200